### ISSUE TITLE
App Crashed When Making Video Call

### ISSUE THREAD

**Issue by Vinayak0-0** [flutter-webrtc/dart-sip-ua#463]
The application getting crashed in linux when making video call.

**To Reproduce**
1. Clone the repo
2. Run the sample code
3. Connect it with asterisk server
4. Now try to make a video call
You will see that app crashed without any errors/warning.

**Expected behavior**
The application shouldn't get crashed

**System Infomation()**
Flutter SDK Version: 3.22.2
OS: Linux Ubuntu 24.04 LTS

**Comment by Ruban-07** [comment]
Yeah! I'm too facing this same issue, anyone know the the solution???
